quantify-basecounter.wdl

```diff
@@ -0,0 +1,24 @@
+import "tasks/biopet.wdl" as biopet
+
+workflow QuantifyBaseCounter {
+    File bamFile
+    String outputDir
+    String sample
+    String stranded
+
+    call biopet.BaseCounter as baseCounter {
+        input:
+            bam = bamFile,
+            outputDir = outputDir,
+            prefix = sample
+    }
+
+    output {
+        File counts =
+            if stranded == "yes" then baseCounter.geneSense
+            else (
+                if stranded == "reverse" then baseCounter.geneAntisense
```

Comment:
This here is incorrect. For stranded there are official 5 possibilities, None, FF, RR FR and RF

If none it should not be sense or antisense. The others I'm not completely sure which is sense and which is antisense.

There is a possibility that 2 of the 4 are not supported by the basecaller, we should take a look at this

Reply:
Every quantification tool I have encountered so far (HTseq-count, STAR quantmode, featurecounts(?), stringtie) only seem to support three strandedness options: None, FR or RF. BaseCounter I'm not sure...

Reply:
Base counter support all 3 in 1 run, no need for an argument. Sense is I thin FR and anti-sense is RF. We should validate this to be sure. None is the sum of both and is also outputted as a files (without sense or anti-sense)

Reply:
Yes, I meant I wasn't sure if it supports FF and RR.

Reply:
Not that I know, but that is only when it's something else then paired-end reads. so mate-pair or solid data can have FF or RR orientation. I think it's safe to ignore those for now